animator/src/animations/rainbow_waterfall.rs

```diff
@@ -17,7 +26,24 @@ impl Animation for RainbowWaterfall {
     fn frame(&mut self, time: f64) -> client::Frame {
         self.points_height
             .iter()
-            .map(|h| client::Color::hsv(h + time, 1.0, 0.5))
+            .map(|h| client::Color::hsv(h * self.parameters.cycles + time, 1.0, 0.5))
             .into()
     }
+
+    fn parameter_schema(&self) -> serde_json::Value {
+        json!({
```

Comment:
Maybe we could add a `"description"` field to the schema? It could be helpful to better understand the parameters, especially if we'd use it in the UI

Reply:
Adding a title and description to every field (also enum option) is definitely something I would like to do. However, I decided to skip it for now and add them when I create appropriate components in the frontend.